### ISSUE TITLE
[Bug] The unexpected opacity of the bar occurs after triggering the Series Highlight.

### ISSUE THREAD

**Issue by Crayon-ShinChan** [VisActor/VChart#2505]
### Version

1.10.2

### Link to Minimal Reproduction

https://www.visactor.io/vchart/demo/interaction/highlight-by-legend?keyword=commonChart

### Steps to Reproduce

It's the component in our document that can interact directly with it.

### Current Behavior

The unexpected opacity of the bar occurs after triggering the Series Highlight. You can check the following video.

https://github.com/VisActor/VChart/assets/38809606/b9fc0d81-93f4-4d00-b8e0-cace9b0df5c8

### Expected Behavior

The bar opacity should change to full when not hovering over any legend.

### Environment

```markdown
- OS: macOS Montery
- Browser: Arc Version 1.35.0 (47750)
- Framework:
```

**Comment by xile611** [comment]
this is caused by the animation of switch states of rect

**Comment by xile611** [comment]
fixed in 1.10.5
